lib/kernel/doc/src/pg.xml

```diff
@@ -84,6 +84,24 @@
       is configured to do so.
     </p>
 
+    <p>If the <seeapp marker="kernel_app#pg_shards_and_metadata">
+      <c>shards_and_metadata</c></seeapp> <c>kernel</c> application
+      parameter is enabled when starting the scope, the server will start with
+      the <c>shards_and_metadata</c> feature enabled. When enabled
+      a member on the form <c>{pid(), Metadata :: any()}</c> can also join
+      or leave the groups.</p>
+
+    <p>In addition, when the feature is enabled, besides the singleton group
+      defined above, the module can also join or leave to a range of shards
```

Comment:
```suggestion
      defined above, `pg` can also join or leave to a range of shards
```
Perhaps it should be `pg` instead of process. Not sure. Module felt wrong though.